Phonebook.Frontend/src/app/pages/page-information/page-information.component.html

```diff
@@ -54,7 +54,7 @@
               Button@@PageInformationComponentSectionFeedbackButton"
             mat-stroked-button
             color="primary"
-            feedbackDrawer
```

Comment:
I think we shouldn't make the feedbackdrawer into a dialogue. Because that's a simple call to action. versus the dialogues which contain information or configuration.

Reply:
if we do this we would have to do this also to call drawers: 
![image](https://user-images.githubusercontent.com/13590797/79965895-e7f36f80-848c-11ea-8600-93283307c20c.png)

Reply:
That's up to you. Should we make an Issue on that and see what others say?

Reply:
@T-Systems-MMS/phonebook-developers

Reply:
> I think we shouldn't make the feedbackdrawer into a dialogue. Because that's a simple call to action. versus the dialogues which contain information or configuration.

+1

Reply:
Fixed with [16ebf01](https://github.com/T-Systems-MMS/phonebook/commit/16ebf01799c355fa1f3a8fd9cc0a8e1aeec62c4f)